modules/xfem/include/efa/EFAElement3D.h

```diff
@@ -31,6 +31,7 @@ class EFAElement3D : public EFAElement
   std::vector<EFAFace *> _faces;
   std::vector<EFAVolumeNode *> _interior_nodes;
   std::vector<std::vector<EFAElement3D *>> _face_neighbors;
+  std::vector<std::vector<std::vector<EFAElement3D *>>> _face_edge_neighbors;
```

Comment:
If you changed this to be of type:
`std::vector<std::vector<std::set<EFAElement3D *>>> _face_edge_neighbors;`
you could avoid the need have nullptr entries and do special things if the size of the last dimension of this is zero. I'm OK with it either way, but that could clean things up a little.

Reply:
Per our discussion, let's leave it as a vector, and change the code so it's initialized with an empty vector rather than a vector of size 1 with a nullptr.